auth/models.go

```diff
@@ -184,6 +190,16 @@ func IsAuthorized(user_id string, orgId string, role string, w http.ResponseWrit
 	var user user.User
 	mapstructure.Decode(userDoc, &user)
 
+	if role == "zuri_admin" {
+		if user.Role == role {
+			return true
+		} else {
+			utils.GetError(errors.New("Access Denied"), http.StatusUnauthorized, w)
+			return false
+		}
```

Comment:
User role suppose be a array, and also declare the list of roles as a constant in models

Reply:
There is no need for that at all, a user has just one role according to the authorisation implementation

Reply:
I think it's better to add it to config file or as a constant in auth/model, so anyone can refresh it as a single source of truth.

Reply:
I think it's better to add it to config file or as a constant in auth/model, so anyone can refrence it as a single source of truth.